python/pyspark/sql/functions.py

```diff
@@ -626,6 +623,10 @@ def mode(col: "ColumnOrName") -> Column:
     :class:`~pyspark.sql.Column`
         the most frequent value in a group.
 
+    Notes
```

Comment:
I am fine with this `Notes`

but does it means all the 150+ functions added in 3.5.0 needs this?

Reply:
That's a good point..